addons/l10n_din5008/report/din5008_report.xml

```diff
@@ -200,5 +185,93 @@
                 </t>
             </xpath>
         </template>
+
+        <template id="report_invoice_document" inherit_id="account.report_invoice_document">
+            <xpath expr="//t[@t-set='forced_vat']" position="after">
+                <t t-set="din5008_document_information">
+                    <div class="information_block" t-if="o and o._name=='account.move'">
```

Comment:
This is already `account.report_invoice_document`, why do you need to still check the `_name`?